File: Tareas/Unidad2/a2253332351_practica03/21_PromedioCalificaciones.cpp
#include <iostream>
using namespace std;

int main() {
    double nota1;
    double nota2;
    double nota3;
    double promedio;

    cout << "Ingrese la primera calificación: ";
    cin >> nota1;
    cout << "Ingrese la segunda calificación: ";
    cin >> nota2;
    cout << "Ingrese la tercera calificación: ";
    cin >> nota3;

    promedio = (nota1 + nota2 + nota3) / 3;

    cout << "El promedio es: " << promedio << endl;

    return 0;
}

File: Tareas/Unidad2/a2253332351_practica03/21_PromedioCalificaciones.java
import java.util.Scanner;

public class PromedioCalificaciones {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double nota1, nota2, nota3, promedio;

          System.out.println("Ingrese la primera calificación: ");
          nota1 = entrada.nextDouble();
          System.out.println("Ingrese la segunda calificación: ");
          nota2 = entrada.nextDouble();
          System.out.println("Ingrese la tercera calificación: ");
          nota3 = entrada.nextDouble();

          promedio = (nota1 + nota2 + nota3) / 3;

          System.out.println("El promedio es: " + promedio);
    }
}
